src/ray/raylet/node_manager.h

```diff
@@ -720,6 +721,9 @@ class NodeManager : public rpc::NodeManagerServiceHandler {
   /// The resources (and specific resource IDs) that are currently available.
   ResourceIdSet local_available_resources_;
   std::unordered_map<ClientID, SchedulingResources> cluster_resource_map_;
+
+  absl::flat_hash_map<JobID, rpc::JobTableData> job_info_cache_;
```

Comment:
Please add a comment here about how this is populated and cleaned up.

Reply:
Done.